torch/_inductor/cudagraph_trees.py

```diff
@@ -381,6 +382,8 @@ def deferred_cudagraphify(inputs: List[InputType]) -> OutputType:
         else:
             log.info("recording cudagraph tree for symint key %s", int_key)
 
+        maybe_warning_due_to_dynamic_shape(fn_cache, int_key)
```

Comment:
I think we should probalby only warn once. otherwise we get spam like this: 

https://github.com/pytorch/pytorch/actions/runs/10270985763/job/28421820609#step:20:3951

wdyt ?